processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2021-12-02 05:32:45.087651614 -0500
+++ memcpu.hex	2021-12-02 05:32:45.026649429 -0500
@@ -99,38 +99,38 @@
 :0400C30003E000084E
 :0401000000000001FA
 :0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
-:04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
+:040102000000005A9F
+:0401030000000051A7
+:0401040000000033C4
+:0401050000000019DD
+:0401060000000050A5
+:0401070000000029CB
+:0401080000000016DD
+:0401090000000015DD
+:04010A000000000CE5
+:04010B000000003EB2
+:04010C000000004BA4
+:04010D0000000047A7
+:04010E00000000539A
+:04010F00000000519B
+:040110000000004D9E
+:0401110000000016D4
+:040112000000000BDE
+:040113000000001DCB
+:0401140000000007E0
+:0401150000000021C5
+:040116000000006382
+:040117000000001BC9
+:04011800000000647F
+:0401190000000042A0
+:04011A000000003DA4
+:04011B0000000020C0
+:04011C0000000001DE
+:04011D0000000036A8
+:04011E0000000004D9
+:04011F000000003D9F
+:0401200000000038A3
+:0401210000000003D7
 :0401220000000007D2
 :0401230000000007D1
 :0401240000000007D0
@@ -145,8 +145,8 @@
 :04012D0000000017B7
 :04012E000000001AB3
 :04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
+:040130000000004487
+:0401310000000020AA
 :0401320000000024A5
 :0401330000000026A2
 :0401340000000026A1
@@ -163,69 +163,18 @@
 :04013F000000005C60
 :040140000000005C5F
 :040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
-:043FFE0000000600B9
+:0401B2000000005CED
+:0401B30000000063E5
+:0401B4000000001B2C
+:0401B50000000064E2
+:0401B60000000064E1
+:0401B7000000004202
+:0401B8000000003D06
+:0401B9000000002022
+:0401BA000000000140
+:0401BB00000000360A
+:0401BC00000000043B
+:0401BD000000003D01
+:0401BE000000003805
+:0401BF000000000339
 :00000001FF
processing file asmFiles/palgorithm.asm (1 of 1)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
--- memsim.hex	2021-12-02 05:36:05.285820586 -0500
+++ memcpu.hex	2021-12-02 05:36:05.208817828 -0500
@@ -99,38 +99,38 @@
 :0400C30003E000084E
 :0401000000000001FA
 :0401010000000040BA
-:0401020000000001F8
-:0401030000000003F5
-:0401040000000004F3
-:0401050000000007EF
-:040106000000000BEA
-:040107000000000CE8
-:0401080000000015DE
-:0401090000000016DC
-:04010A0000000016DB
-:04010B0000000019D7
-:04010C000000001BD4
-:04010D000000001DD1
-:04010E0000000020CD
-:04010F0000000021CB
-:0401100000000029C2
-:0401110000000033B7
-:0401120000000036B3
-:0401130000000038B0
-:040114000000003DAA
-:040115000000003DA9
-:040116000000003EA7
-:0401170000000042A2
-:04011800000000479C
-:040119000000004B97
-:04011A000000004D94
-:04011B000000005090
-:04011C00000000518E
-:04011D00000000518D
-:04011E00000000538A
-:04011F000000005A82
-:040120000000006378
-:040121000000006476
+:040102000000005A9F
+:0401030000000051A7
+:0401040000000033C4
+:0401050000000019DD
+:0401060000000050A5
+:0401070000000029CB
+:0401080000000016DD
+:0401090000000015DD
+:04010A000000000CE5
+:04010B000000003EB2
+:04010C000000004BA4
+:04010D0000000047A7
+:04010E00000000539A
+:04010F00000000519B
+:040110000000004D9E
+:0401110000000016D4
+:040112000000000BDE
+:040113000000001DCB
+:0401140000000007E0
+:0401150000000021C5
+:040116000000006382
+:040117000000001BC9
+:04011800000000647F
+:0401190000000042A0
+:04011A000000003DA4
+:04011B0000000020C0
+:04011C0000000001DE
+:04011D0000000036A8
+:04011E0000000004D9
+:04011F000000003D9F
+:0401200000000038A3
+:0401210000000003D7
 :0401220000000007D2
 :0401230000000007D1
 :0401240000000007D0
@@ -145,8 +145,8 @@
 :04012D0000000017B7
 :04012E000000001AB3
 :04012F0000000020AC
-:0401300000000020AB
-:0401310000000023A7
+:040130000000004487
+:0401310000000020AA
 :0401320000000024A5
 :0401330000000026A2
 :0401340000000026A1
@@ -163,69 +163,18 @@
 :04013F000000005C60
 :040140000000005C5F
 :040141000000006456
-:04018000000000017A
-:040181000000000377
-:040182000000000475
-:040183000000000771
-:040184000000000770
-:04018500000000076F
-:04018600000000076E
-:04018700000000086C
-:04018800000000086B
-:04018900000000086A
-:04018A000000000968
-:04018B000000000A66
-:04018C000000000B64
-:04018D000000000C62
-:04018E000000000F5E
-:04018F00000000105C
-:04019000000000115A
-:040191000000001555
-:040192000000001653
-:040193000000001652
-:040194000000001750
-:04019500000000194D
-:040196000000001A4B
-:040197000000001B49
-:040198000000001D46
-:040199000000002042
-:04019A000000002041
-:04019B000000002040
-:04019C00000000213E
-:04019D00000000233B
-:04019E000000002439
-:04019F000000002636
-:0401A0000000002635
-:0401A1000000002832
-:0401A2000000002930
-:0401A3000000002F29
-:0401A4000000003027
-:0401A5000000003224
-:0401A6000000003322
-:0401A700000000361E
-:0401A800000000381B
-:0401A9000000003D15
-:0401AA000000003D14
-:0401AB000000003E12
-:0401AC00000000410E
-:0401AD00000000420C
-:0401AE00000000420B
-:0401AF000000004309
-:0401B0000000004407
-:0401B1000000004703
-:0401B2000000004BFE
-:0401B3000000004DFB
-:0401B40000000050F7
-:0401B50000000051F5
-:0401B60000000051F4
-:0401B70000000053F1
-:0401B80000000059EA
-:0401B9000000005AE8
-:0401BA000000005BE6
-:0401BB000000005CE4
-:0401BC000000005CE3
-:0401BD0000000063DB
-:0401BE0000000064D9
-:0401BF0000000064D8
-:043FFE0000000600B9
+:0401B2000000005CED
+:0401B30000000063E5
+:0401B4000000001B2C
+:0401B50000000064E2
+:0401B60000000064E1
+:0401B7000000004202
+:0401B8000000003D06
+:0401B9000000002022
+:0401BA000000000140
+:0401BB00000000360A
+:0401BC00000000043B
+:0401BD000000003D01
+:0401BE000000003805
+:0401BF000000000339
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/palgorithm.asm (1 of 1)
processing file asmFiles/palgorithm.asm (1 of 1)
processing file asmFiles/palgorithm.asm (1 of 1)
